src/redux/stores/nftDetail/nftDetailSlice.ts

```diff
@@ -0,0 +1,86 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { CollectionToken, CollectionTokenAttribute } from '../../../entities/CollectionToken/CollectionToken';
+import { fetchNftMeta } from './nftDetailApi';
+
+export interface EventLog {
+  from?: string;
+  to?: string;
+  tokenId: number;
+  type: string;
+}
+
+export interface Error {
```

Comment:
This will be confused with the javascript Error object. In airswap-web we use AppError with enums for each error so it's more easy to translate.

Reply:
Name it `NftDetailError` for now and we will do an Error refactor later.